model/schema/allele.yaml

```diff
@@ -1056,9 +1059,12 @@ slots:
     range: AlleleNoteSlotAnnotation
 
   allele_note_dtos:
+    is_a: note_dtos
```

Comment:
Something for a separate PR, but I think we need to revisit the modelling of this.  We have evidence inherited from the `SlotAnnotation` but also evidence on the `Note` object.  I would have thought that we want to have a similar approach to that of `NameSlotAnnotation` whereby we would have a core NoteSlotAnnotation class the has the `free_text` and `note_type` slots that are currently in `Note`, then the range of `allele_note_dtos` would be `NoteSlotAnnotationDTO`.

Reply:
Yes I agree - it is inconsistent.